ProjectSettings/EditorBuildSettings.asset

```diff
@@ -6,30 +6,15 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/MunroHoberman/WerewolfReviversAssets/Scenes/WerewolfReviversGame.unity
-    guid: b5c6fcb21457310409a71b31476fa441
+    path: Assets/Mariana /Scenes/FirstSceneLoad.unity
+    guid: 7819e91666d2a44b5973068a5daf0ca9
   - enabled: 1
-    path: Assets/MunroHoberman/WerewolfReviversAssets/Scenes/WerewolfReviversWin.unity
-    guid: 375fa8c0254bef2468e5ea2ff90c57ea
+    path: Assets/Mariana /Scenes/Game.unity
+    guid: 4d90912d1b7b747ca9313575e79c9263
   - enabled: 1
-    path: Assets/MunroHoberman/WerewolfReviversAssets/Scenes/WerewolfReviversLose.unity
-    guid: 6cabc1914c9b0c442a4890889ec126cd
+    path: Assets/Mariana /Scenes/EndBad.unity
```

Comment:
Scene changes are fine! You had to update build settinsg to get this to work but it will have an issue with people down the road.